services/web/server/tests/integration/01/test_exporter_requests_handlers.py

```diff
@@ -0,0 +1,236 @@
+    empty_project_data,
+)
+from servicelib.aiohttp.application import create_safe_application
+from simcore_service_webserver._constants import X_PRODUCT_NAME_HEADER
+from simcore_service_webserver._meta import API_VTAG
+from simcore_service_webserver.application import (
+    setup_exporter,
+    setup_login,
+    setup_projects,
+    setup_resource_manager,
+    setup_rest,
+    setup_security,
+    setup_session,
+    setup_socketio,
+    setup_users,
+)
+from simcore_service_webserver.application_settings import setup_settings
+from simcore_service_webserver.db import setup_db
+from simcore_service_webserver.exporter import settings as exporter_settings
+from simcore_service_webserver.exporter.formatter.archive import get_sds_archive_path
+from simcore_service_webserver.projects.project_models import ProjectDict
+from yarl import URL
+
+pytest_simcore_core_services_selection = [
+    "migration",
+    "postgres",
+    "redis",
+]
+
+_logger = logging.getLogger(__name__)
+
+
+CURRENT_DIR = (
+    (Path(sys.argv[0] if __name__ == "__main__" else __file__) / ".." / "..")
+    .resolve()
+    .parent
+)
+
+
+async def _new_project(
+    client: TestClient,
+    user: UserInfoDict,
+    product_name: str,
+    project_template_file: Path,
+) -> ProjectDict:
+    """returns a project for the given user"""
+    project_data = empty_project_data()
+
+    assert client.app
+    return await create_project(
+        client.app,
+        project_data,
+        user["id"],
+        product_name=product_name,
+        default_project_json=project_template_file,
+    )
+
+
+def _get_files_in_zip(zip_path: Path) -> set[str]:
+    return {x.filename for x in ZipFile(zip_path).infolist()}
+
+
+def _get_fake_template_projects() -> set[Path]:
+    template_dir = CURRENT_DIR / "data"
+    assert template_dir.exists()
+    return set(template_dir.rglob("fake-template-projects*"))
+
+
+@pytest.fixture(params=_get_fake_template_projects())
+def template_path(request: FixtureRequest) -> Path:
+    return request.param
+
+
+@pytest.fixture
+def product_name() -> str:
+    return "osparc"
+
+
+@pytest.fixture
+def client(
+    docker_swarm: None,
+    event_loop: asyncio.AbstractEventLoop,
+    aiohttp_client: Callable,
+    app_config: dict,
+    monkeypatch_setenv_from_app_config: Callable,
+) -> Iterable[TestClient]:
+    # test config & env vars ----------------------
+    cfg = deepcopy(app_config)
+    assert cfg["rest"]["version"] == API_VTAG
+    assert cfg["rest"]["enabled"]
+
+    cfg["projects"]["enabled"] = True
+    cfg["director"]["enabled"] = True
+    cfg["exporter"]["enabled"] = True
+
+    monkeypatch_setenv_from_app_config(cfg)
+
+    # app setup ----------------------------------
+    app = create_safe_application(cfg)
+
+    # activates only security+restAPI sub-modules
+    assert setup_settings(app)
+    assert (
+        exporter_settings.get_plugin_settings(app) is not None
+    ), "Should capture defaults"
+
+    setup_db(app)
```

Comment:
if possible adapt the test to the new approach i.e. override `app_environment` fixture with the environs you need (that will enable/disable the plugins you want) and let the base fixture to init the app for you

You can find some examples if you search `def app_environment`.  Here is a nice example

https://github.com/ITISFoundation/osparc-simcore/blob/a7ab0fbf57bd8f9ff36121d740e96a8f1dac64e6/services/web/server/tests/unit/isolated/test_studies_dispatcher_projects_permalinks.py#L28-L68

Reply:
Did some experiments with it. Did not succeed in using `create_application()` with above suggestion.